test/common/types.go

```diff
@@ -36,3 +47,8 @@ type prometheusAPIResponse struct {
 	Error     string                 `json:"error"`
 	Warnings  []string               `json:"warnings,omitempty"`
 }
+
+type SubscriptionCheck struct {
+	Name      string
+	Namespace string
+}
```

Comment:
@KevFan As I mention in @davidffrench PR [1] in my opinion types should remain in the test file, but whatever we choose it would be good to get a common agreement, wdyt?

1. https://github.com/integr8ly/integreatly-operator/pull/527/files#r394928114

Reply:
Yup, shoulds good to me, I'll update the test to match whatever is decided if necessary :+1: